app/controllers/v0/messages_controller.rb

```diff
@@ -2,8 +2,9 @@
 module V0
   class MessagesController < HealthcareMessagingController
     def index
-      folder_id = params[:folder_id].try(:to_i)
-      response = client.get_folder_messages(folder_id, 1, 100)
+      pp = pagination_params
+      # TODO: convert to a hash arg once sm gem is moved over.
+      response = client.get_folder_messages(pp[:folder_id], pp[:page], pp[:per_page], pp[:all])
```

Comment:
ok, so eventually i would like for us to look for a way to change this so that all the pagination wierdness is handled in gem similar to what we do for a collection, but we can revisit that later.